### ISSUE TITLE
Problem with arrows and tooltips

### ISSUE THREAD

**Issue by remontees** [Orange-OpenSource/Orange-Boosted-Angular#262]
Hello,

When using Boosted v5.1 and ng-boosted v10.0.0, the tooltips are not displayed correctly (especially the arrow at the edge of the tooltip).

Best regards

EDIT: In Bootstrap v5.1, the `.arrow` class has changed to `.tooltip-arrow`. ;-)

**Comment by Lausselloic** [comment]
Hello, boosted 5 is not supported currently on ng-boosted. ng-boosted rely on ng-bootstrap that is still in v11-beta to support bootstrap v5 https://ng-bootstrap.github.io/#/home

**Comment by Lausselloic** [comment]
Oh my bad, ng-boosted v10 is annouced to be compliant with boosted v5.
From what I see in the documentation left and right tooltip are not well displayed but to and bottom are okay.

Are you talking about left/right tooltip? Or you also have a problem with top/bottom?

**Comment by remontees** [comment]
> 
> 
> Oh my bad, ng-boosted v10 is annouced to be compliant with boosted v5. From what I see in the documentation left and right tooltip are not well displayed but to and bottom are okay.
> 
> Are you talking about left/right tooltip? Or you also have a problem with top/bottom?

Hello, ofc it is announced to be compliant with Boosted v5.1.

I have this issue with top/bottom tooltips (as I said in my first post, I am quite sure that it is a problem with a renamed class (`.arrow` to `.tooltip-arrow`).

**Comment by Lausselloic** [comment]
Hello @remontees I've update the documentation for left and right tooltips. But nothing changes for top/bottom.

Classname are correct in ng-boosted : 
![image](https://user-images.githubusercontent.com/11272238/141770082-634ed112-c17d-400f-9fbd-8f58400e5d60.png)

Did you import ng-boostrap in your project maybe? or boosted? If yes you need to remove them and rely only on ng-boosted.

**Comment by remontees** [comment]
> 
> 
> Hello @remontees I've update the documentation for left and right tooltips. But nothing changes for top/bottom.
> 
> Classname are correct in ng-boosted : ![image](https://user-images.githubusercontent.com/11272238/141770082-634ed112-c17d-400f-9fbd-8f58400e5d60.png)
> 
> Did you import ng-boostrap in your project maybe? or boosted? If yes you need to remove them and rely only on ng-boosted.

Hello,
Thanks for your answer.
I just tried but it doesn't work. Indeed, we had manually installed Boosted and ng-bootstrap, but we paid attention to install the versions compatible with the 10.0.0 version of ng-boosted.
Could it be about the fact that a tooltip must be set relative to a div element (and not a button)?

**Comment by Lausselloic** [comment]
did you install ng-bootstrap 11.0.0-beta.2 ? because it's the only one that is compliant with boosted v5

On ng-boosted tooltip are relative to a button

**Comment by remontees** [comment]
> did you install ng-bootstrap 11.0.0-beta.2 ? because it's the only one that is compliant with boosted v5
> 
> On ng-boosted tooltip are relative to a button

No, I installed ng-boostrap 10.0.0. If it is true that only v11 is compatible with boosted v5, why can we read this on the ng-boosted website : https://ng-boosted.orange.com/#/start
![Capture](https://user-images.githubusercontent.com/1620974/142008097-5e8eac17-94ea-4015-97a1-cea2205c56be.PNG)

Moreover, unless I'm misunderstanding, we already discussed this issue 11 days ago here, and you already acknowledged that ng-boosted v10 was compatible with boosted v5.
Would it be possible to have a clear information?

**Comment by Lausselloic** [comment]
Ng-boosted already include ng-bootstrap AND Boosted so fi you install ng-boosted you don't need to install boosted and ng-bootstrap into your project.

Ng-boosted v10 include ng-bootstrap-beta.2 wich is compliant with boosted v5
